.npmrc

```diff
@@ -0,0 +1,5 @@
+color=false
+engine-strict=true
+fund=false
+progress=true
+unicode=false
```

Comment:
these changes make npm logs easier to read on non-unicode-enabled terminals (like bamboo's)